src/App.js

```diff
@@ -84,6 +85,16 @@ class App extends Component {
               </Link>
             </li>
           </div>
+
+          {/* I think this goes here? -Josten */}
```

Comment:
For now; we'll meet and refine navigation, but probably will have that as a tech debt story for the next sprint.